config.rb.sample

```diff
@@ -6,12 +6,17 @@ $new_discovery_url='https://discovery.etcd.io/new'
 #if File.exists?('user-data') && ARGV[0].eql?('up')
 #  require 'open-uri'
 #  require 'yaml'
-# 
+#
 #  token = open($new_discovery_url).read
```

Comment:
ah, a fellow python person, doing this and testing (ruby syntax is "stuff?size=#{$num_instances}")